server/src/services/item.service.ts

```diff
@@ -1,13 +1,29 @@
 import Item from '../models/item.model';
 import ItemCategory from '../models/itemCategory.model';
-import SORT_TYPE from '../utils/SortType';
 import { Op, Transaction, WhereOptions } from 'sequelize';
 import ItemOption from '../models/itemOption.model';
 import ItemError, { ItemErrorType } from '../errors/item.error';
+import esClient from '../config/elasticsearch';
 import ItemTag from '../models/itemTag.model';
 import GetItemsRequest from '../requests/item/getItems.request';
 
 class ItemService {
+  async _searchToElasticSearch(keyword: string) {
+    const result = await esClient.search({
+      index: 'store_item',
+      size: 10,
+      body: {
+        query: {
+          match: {
+            title: keyword,
+          },
+        },
+      },
+    });
+
+    return result;
```

Comment:
```suggestion
    return result.hits.hits;
```
사용처에서 보면 다 `result.hits.hits` 를 쓰던데, 아예 그렇게 내보내는 건 어떨까요?

Reply:
hits가 아닌 따른것들을 혹시 사용할 수 있을까 해서 result 만 보냈는데 해당 방법도 좋아보이네요 :)